### PR TITLE
connections api master page

### PR DESCRIPTION
**Pull request by rani2655**

### CODE REVIEW

File: modules/ROOT/pages/connections.adoc [hunk 1]
```diff
@@ -0,0 +1,2969 @@
...
+The connection API endpoints allow you to create, edit, delete, export, and query data connections on a ThoughtSpot cluster. See, the following pages for the REST API endpoints:
+
+* https://developers.thoughtspot.com/docs/connections-api[REST APIs v1, window=_blank]
+* https://developers.thoughtspot.com/docs/rest-apiv2-reference#_connections[REST APIs v2, window=_blank]
+
+=== Required permissions
+
+You must have data management permissions to create, edit, delete or query data connection objects. Make sure your user account has the `DATAMANAGEMENT` privilege.
+
+If https://developers.thoughtspot.com/docs/rbac[Role-Based Access Control (RBAC), window=_blank] is enabled, the `CAN_CREATE_OR_EDIT_CONNECTIONS` privilege is required.
```
Comment (ShashiSubramanya) [inline modules/ROOT/pages/connections.adoc:83]: Change "CAN_CREATE_OR_EDIT_CONNECTIONS" to "Can create/edit Connections"

File: modules/ROOT/pages/connections.adoc [hunk 2]
```diff
@@ -0,0 +1,2969 @@
...
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-teradata[Teradata]
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/connections-trino[Trino]
+
+
+
+This article describes the procedures for setting and managing data connections through the REST APIs. For information about how to set up a data connection in the ThoughtSpot UI, see link:https://cloud-docs.thoughtspot.com/admin/ts-cloud/embrace.html[Data Connections in ThoughtSpot, window=_blank].
+
+== Data Connection APIs
+The connection API endpoints allow you to create, edit, delete, export, and query data connections on a ThoughtSpot cluster. See, the following pages for the REST API endpoints:
```
Comment (ShashiSubramanya) [inline modules/ROOT/pages/connections.adoc:74]: For more information about the API endpoints, see the following pages:

File: modules/ROOT/pages/connections.adoc [hunk 3]
```diff
@@ -0,0 +1,2969 @@
...
+
+* https://developers.thoughtspot.com/docs/connections-api[REST APIs v1, window=_blank]
+* https://developers.thoughtspot.com/docs/rest-apiv2-reference#_connections[REST APIs v2, window=_blank]
+
```
Comment (ShashiSubramanya) [inline modules/ROOT/pages/connections.adoc:78]: Change the first link to "xref:connections-api.adoc[REST API v1].

Comment (rani2655) [reply modules/ROOT/pages/connections.adoc:78]: Done.

File: modules/ROOT/pages/connections.adoc [hunk 4]
```diff
@@ -0,0 +1,2969 @@
...
+== Data catalog integration
+
+ThoughtSpot supports integration with only one data catalog tool at a time. You must have `CAN_CREATE_CATALOG` permissions to establish a data catalog connection.
+To understand about the data catalog tools supported by ThoughtSpot, see the following pages:
+
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/catalog-integration[Alation, window=_blank]
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/catalog-integration-atlan[Atlan, window=_blank]
+* https://docs.thoughtspot.com/cloud/10.0.0.cl/catalog-integration-collibra[Collibra, window=_blank]
+
```
[the remark this answers is not in the document]
Comment (rani2655) [reply modules/ROOT/pages/connections.adoc:610]: Done.

**Pull request merged**